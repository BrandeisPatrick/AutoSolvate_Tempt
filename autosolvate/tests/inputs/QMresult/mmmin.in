Minimize
&cntrl
imin=1,
ntx=1,
maxcyc=2000,
ncyc=1000,
ntpr=100,
ntwx=0,
cut=8.0,
/
