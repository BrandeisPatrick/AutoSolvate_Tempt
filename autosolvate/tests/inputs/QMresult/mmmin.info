

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -4.1292E+04     2.0265E-01     1.1176E+01     FE          1

 BOND    =       40.2151  ANGLE   =      327.7786  DIHED      =        2.9546
 VDWAALS =    -7485.4811  EEL     =    -7116.5676  HBOND      =        0.0000
 1-4 VDW =      -96.3081  1-4 EEL =   -26964.7811  RESTRAINT  =        0.0000
