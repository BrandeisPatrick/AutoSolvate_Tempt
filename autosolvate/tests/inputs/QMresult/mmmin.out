
          -------------------------------------------------------
          Amber 22 SANDER                              2022
          -------------------------------------------------------

| Run on 03/19/2024 at 15:21:52

|   Executable path: sander
| Working directory: /home/sunnyxun/projects/new_project_AutoMCPB/RedoxPotential/Fe/plus2/FFgen
|          Hostname: node7

  [-O]verwriting output

File Assignments:
|  MDIN: mmmin.in                                                              
| MDOUT: mmmin.out                                                             
|INPCRD: Fe_plus2_solvated.inpcrd                                              
|  PARM: Fe_plus2_solvated.prmtop                                              
|RESTRT: mm.ncrst                                                              
|  REFC: refc                                                                  
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: mden                                                                  
| MDCRD: mdcrd                                                                 
|MDINFO: mmmin.info                                                            
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

Minimize                                                                       
&cntrl                                                                         
imin=1,                                                                        
ntx=1,                                                                         
maxcyc=2000,                                                                   
ncyc=1000,                                                                     
ntpr=100,                                                                      
ntwx=0,                                                                        
cut=8.0,                                                                       
/                                                                              

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
 getting new box info from bottom of inpcrd
|  INFO: Old style inpcrd file read

| peek_ewald_inpcrd: Box info found
|Largest sphere to fit in unit cell has radius =    28.000
| New format PARM file being parsed.
| Version =    1.000 Date = 03/19/24 Time = 15:21:12
 NATOM  =   10953 NTYPES =       8 NBONH =    5469 MBONA  =    3675
 NTHETH =   10938 MTHETA =    1890 NPHIH =    5547 MPHIA  =     144
 NHPARM =       0 NPARM  =       0 NNB   =   29423 NRES   =    1821
 NBONA  =    3675 NTHETA =    1890 NPHIA =     144 NUMBND =      26
 NUMANG =      66 NPTRA  =       6 NATYP =      16 NPHB   =       0
 IFBOX  =       1 NMXRS  =      20 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0


|     Memory Use     Allocated
|     Real             1632430
|     Hollerith          34682
|     Integer           629189
|     Max Pairs        3651000
|     nblistReal        131436
|     nblist Int        695998
|       Total            33353 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals

     BOX TYPE: RECTILINEAR

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       0

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       2, ntpr    =     100, ntrx    =       1, ntwr    =       1
     iwrap   =       0, ntwx    =       0, ntwv    =       0, ntwe    =       0
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       1, ntb     =       1, igb     =       0, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     =   8.00000, intdiel =   1.00000

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       0

Energy minimization:
     maxcyc  =    2000, ncyc    =    1000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

Ewald parameters:
     verbose =       0, ew_type =       0, nbflag  =       1, use_pme =       1
     vdwmeth =       1, eedmeth =       1, netfrc  =       0
     Box X =   56.000   Box Y =   56.000   Box Z =   56.000
     Alpha =   90.000   Beta  =   90.000   Gamma =   90.000
     NFFT1 =   60       NFFT2 =   60       NFFT3 =   60
     Cutoff=    8.000   Tol   =0.100E-04
     Ewald Coefficient =  0.34864
     Interpolation order =    4

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps

 Number of triangulated 3-point waters found:        0

     Sum of charges from parm topology file =   0.01815169
     Assuming uniform neutralizing plasma

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------

 ---------------------------------------------------
 APPROXIMATING switch and d/dx switch using CUBIC SPLINE INTERPOLATION
 using   5000.0 points per unit in tabled values
 TESTING RELATIVE ERROR over r ranging from 0.0 to cutoff
| CHECK switch(x): max rel err =   0.2738E-14   at   2.422500
| CHECK d/dx switch(x): max rel err =   0.8332E-11   at   2.782960
 ---------------------------------------------------
| Local SIZE OF NONBOND LIST =    1491030
| TOTAL SIZE OF NONBOND LIST =    1491030


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1       2.8150E+05     1.1297E+04     1.1749E+06     Cl-     10952

 BOND    =     3294.7002  ANGLE   =     2613.5140  DIHED      =        0.1452
 VDWAALS =   302933.2342  EEL     =       31.6185  HBOND      =        0.0000
 1-4 VDW =      -78.6441  1-4 EEL =   -27297.2456  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    100      -2.4885E+04     5.3746E+00     7.3231E+01     C2       8829

 BOND    =      559.9503  ANGLE   =     1104.0680  DIHED      =        0.9268
 VDWAALS =     1919.1954  EEL     =    -1249.0629  HBOND      =        0.0000
 1-4 VDW =      -85.0465  1-4 EEL =   -27135.2625  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    200      -3.0466E+04     4.5963E+00     7.3120E+01     C2       8829

 BOND    =      333.0639  ANGLE   =      620.7954  DIHED      =        1.3945
 VDWAALS =    -2257.6666  EEL     =    -1995.3973  HBOND      =        0.0000
 1-4 VDW =      -90.5396  1-4 EEL =   -27077.5084  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    300      -3.2888E+04     8.3369E-01     6.1997E+00     Cl-     10952

 BOND    =      161.9690  ANGLE   =      488.8349  DIHED      =        1.5973
 VDWAALS =    -3826.3580  EEL     =    -2571.6875  HBOND      =        0.0000
 1-4 VDW =      -92.3935  1-4 EEL =   -27050.1507  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    400      -3.4261E+04     1.3476E+00     2.0382E+01     C2       8829

 BOND    =      137.3439  ANGLE   =      435.3607  DIHED      =        1.6881
 VDWAALS =    -4668.0577  EEL     =    -3039.1672  HBOND      =        0.0000
 1-4 VDW =      -93.2678  1-4 EEL =   -27034.9845  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -3.5200E+04     1.0891E+00     2.4774E+01     C7         50

 BOND    =      116.4961  ANGLE   =      408.9000  DIHED      =        1.7858
 VDWAALS =    -5187.4654  EEL     =    -3419.7128  HBOND      =        0.0000
 1-4 VDW =      -93.8026  1-4 EEL =   -27026.4635  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    600      -3.5899E+04     8.8862E-01     4.0420E+01     C7         50

 BOND    =      101.9736  ANGLE   =      392.6215  DIHED      =        1.8800
 VDWAALS =    -5539.3183  EEL     =    -3743.8605  HBOND      =        0.0000
 1-4 VDW =      -93.9728  1-4 EEL =   -27018.6377  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    700      -3.6444E+04     4.7124E-01     1.0105E+01     C7         50

 BOND    =       91.3917  ANGLE   =      381.2447  DIHED      =        2.0139
 VDWAALS =    -5792.3919  EEL     =    -4019.1350  HBOND      =        0.0000
 1-4 VDW =      -94.3310  1-4 EEL =   -27013.2434  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    800      -3.6893E+04     4.3334E-01     9.7213E+00     C7         50

 BOND    =       84.6761  ANGLE   =      373.5643  DIHED      =        2.1495
 VDWAALS =    -5987.1770  EEL     =    -4263.5342  HBOND      =        0.0000
 1-4 VDW =      -94.4424  1-4 EEL =   -27008.6547  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    900      -3.7270E+04     3.8747E-01     7.4373E+00     C7         50

 BOND    =       79.7265  ANGLE   =      367.4281  DIHED      =        2.2991
 VDWAALS =    -6140.7826  EEL     =    -4479.3265  HBOND      =        0.0000
 1-4 VDW =      -94.6483  1-4 EEL =   -27004.8161  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -3.7580E+04     1.3201E+00     6.8422E+01     C8         51

 BOND    =       84.1958  ANGLE   =      363.1007  DIHED      =        2.4631
 VDWAALS =    -6264.0687  EEL     =    -4669.5539  HBOND      =        0.0000
 1-4 VDW =      -94.9659  1-4 EEL =   -27001.5454  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1100      -3.9415E+04     8.2712E-01     3.5633E+01     N1         33

 BOND    =       61.6869  ANGLE   =      347.2872  DIHED      =        4.7228
 VDWAALS =    -6889.0829  EEL     =    -5860.3512  HBOND      =        0.0000
 1-4 VDW =      -97.4015  1-4 EEL =   -26982.0842  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1200      -3.9738E+04     5.1340E-01     2.2241E+01     C5         48

 BOND    =       54.8850  ANGLE   =      340.9234  DIHED      =        4.1444
 VDWAALS =    -6998.2232  EEL     =    -6063.1547  HBOND      =        0.0000
 1-4 VDW =      -97.1935  1-4 EEL =   -26978.9055  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1300      -4.0001E+04     5.1242E-01     2.0379E+01     C2         45

 BOND    =       52.3831  ANGLE   =      339.3118  DIHED      =        3.9636
 VDWAALS =    -7080.2817  EEL     =    -6242.9202  HBOND      =        0.0000
 1-4 VDW =      -97.0817  1-4 EEL =   -26976.1269  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1400      -4.0196E+04     5.4774E-01     2.3915E+01     H7         61

 BOND    =       54.6849  ANGLE   =      337.4735  DIHED      =        4.1271
 VDWAALS =    -7141.1880  EEL     =    -6378.1313  HBOND      =        0.0000
 1-4 VDW =      -97.5039  1-4 EEL =   -26975.0438  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -4.0406E+04     4.0758E-01     1.3608E+01     C9         32

 BOND    =       49.0305  ANGLE   =      333.8321  DIHED      =        3.6601
 VDWAALS =    -7204.8942  EEL     =    -6518.3761  HBOND      =        0.0000
 1-4 VDW =      -96.8326  1-4 EEL =   -26972.6172  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1600      -4.0862E+04     3.0248E-01     1.4689E+01     H          54

 BOND    =       43.5241  ANGLE   =      332.1634  DIHED      =        3.3565
 VDWAALS =    -7330.2525  EEL     =    -6845.7973  HBOND      =        0.0000
 1-4 VDW =      -95.3457  1-4 EEL =   -26969.2331  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1700      -4.0968E+04     2.5558E-01     9.7017E+00     C6         49

 BOND    =       41.9316  ANGLE   =      329.8148  DIHED      =        3.4385
 VDWAALS =    -7375.9146  EEL     =    -6904.8788  HBOND      =        0.0000
 1-4 VDW =      -95.6904  1-4 EEL =   -26966.6987  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1800      -4.1066E+04     3.9702E-01     2.0399E+01     FE          1

 BOND    =       41.7634  ANGLE   =      330.3857  DIHED      =        3.1878
 VDWAALS =    -7410.2326  EEL     =    -6969.8985  HBOND      =        0.0000
 1-4 VDW =      -95.2802  1-4 EEL =   -26965.8903  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1900      -4.1160E+04     6.0936E-01     2.6233E+01     C6         49

 BOND    =       42.1631  ANGLE   =      330.3551  DIHED      =        3.9008
 VDWAALS =    -7442.8853  EEL     =    -7033.7835  HBOND      =        0.0000
 1-4 VDW =      -94.8933  1-4 EEL =   -26964.5842  RESTRAINT  =        0.0000


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -4.1292E+04     2.0265E-01     1.1176E+01     FE          1

 BOND    =       40.2151  ANGLE   =      327.7786  DIHED      =        2.9546
 VDWAALS =    -7485.4811  EEL     =    -7116.5676  HBOND      =        0.0000
 1-4 VDW =      -96.3081  1-4 EEL =   -26964.7811  RESTRAINT  =        0.0000


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -4.1292E+04     2.0265E-01     1.1176E+01     FE          1

 BOND    =       40.2151  ANGLE   =      327.7786  DIHED      =        2.9546
 VDWAALS =    -7485.4811  EEL     =    -7116.5676  HBOND      =        0.0000
 1-4 VDW =      -96.3081  1-4 EEL =   -26964.7811  RESTRAINT  =        0.0000

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|    Read coords time           0.01 ( 0.01% of Total)
|                Build the list             0.69 (93.37% of List )
|                Other                      0.05 ( 6.63% of List )
|             List time                  0.74 ( 0.71% of Nonbo)
|                   Short_ene time            68.12 (98.74% of Direc)
|                   Other                      0.87 ( 1.26% of Direc)
|                Direct Ewald time         68.99 (67.16% of Ewald)
|                Adjust Ewald time          1.50 ( 1.46% of Ewald)
|                   Fill Bspline coeffs        1.23 ( 3.86% of Recip)
|                   Fill charge grid           3.29 (10.31% of Recip)
|                   Scalar sum                 6.36 (19.94% of Recip)
|                   Grad sum                   6.09 (19.10% of Recip)
|                   FFT time                  14.91 (46.79% of Recip)
|                Recip Ewald time          31.88 (31.03% of Ewald)
|                Virial junk                0.09 ( 0.09% of Ewald)
|                Other                      0.26 ( 0.25% of Ewald)
|             Ewald time               102.73 (99.28% of Nonbo)
|          Nonbond force            103.47 (95.63% of Force)
|          Bond/Angle/Dihedral        4.65 ( 4.30% of Force)
|          Other                      0.07 ( 0.06% of Force)
|       Force time               108.19 (100.0% of Runmd)
|    Runmd Time               108.19 (71.84% of Total)
|    Other                     42.40 (28.15% of Total)
| Total time               150.60 (100.0% of ALL  )

| Number of list builds   :         12

| Highest rstack allocated:     768433
| Highest istack allocated:      10953
|           Job began  at 15:21:52.530  on 03/19/2024
|           Setup done at 15:21:52.718  on 03/19/2024
|           Run   done at 15:24:23.129  on 03/19/2024
|     wallclock() was called   76072 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
